cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -1585,6 +1586,7 @@ class CallInstruction extends Instruction {
   /**
    * Gets the argument at the specified index.
    */
+  pragma[noinline]
```

Comment:
I feel more confident claiming that the other predicate, `getPositionalArgumentOperand`, can do without `pragma[noinline]`. The slow version of `definitionByReferenceNodeFromArgument` has `getPositionalArgumentOperand` in its tuple counts, so that predicate seems to not have been inlined.

In any case, I don't think it was strictly necessary to put the pragma on both predicates. I did it anyway because I worried we'd otherwise have a similar performance problem the next time we try and do a slightly different thing with these predicates. Joining on parameter indexes is always risky, and I wanted to reduce that risk.